test/self_driving/modeling_test.cpp

```diff
@@ -83,7 +83,7 @@ TEST_F(ModelServerTest, PipelineTest) {
   auto result = ms_manager->DoInference("OP_INTEGER_PLUS_OR_MINUS", save_path, features);
   ASSERT_TRUE(result.second);
   ASSERT_EQ(result.first.size(), features.size());
-  result = ms_manager->DoInference("OP_DECIMAL_COMPARE", save_path, features);
+  result = ms_manager->DoInference("OP_REAL_COMPARE", save_path, features);
```

Comment:
nit: in fact, can you actually use `OperatingUnitUtil::ExecutionOperatingUnitTypeToString` to convert `ExecutionOperatingUnitType::OP_REAL_COMPARE` to string? I think this would be a bit more stable against changes. The same applies to other valid operating unit types in this test.

Reply:
Oh yes, I did not know this function exist. This is the right way.